### DISCUSSION TITLE
请问为什么不能网页全屏了

### DISCUSSION THREAD

**Discussion by dabaicaicaii** [the1812/Bilibili-Evolved#3699] [提问 / Q&A]
![微信图片_20220922203604](https://user-images.githubusercontent.com/114159738/191750646-becb9e39-8d1d-4916-85dc-204c4d85e77e.png)
![微信图片_20220922203624](https://user-images.githubusercontent.com/114159738/191750658-7ba05bbe-ad1a-4b37-ba9a-802006b93756.png)
点击网页全屏和宽屏没有变化，只有关闭脚本才可以

**Comment by the1812** [comment]
检查下是哪个功能影响的, 开关脚本范围太大了
